pkg/providers/imagefamily/types.go

```diff
@@ -25,17 +28,60 @@ import (
 const (
 	AKSUbuntuPublicGalleryURL     = "AKSUbuntu-38d80f77-467a-481f-a8d4-09b6d4220bd2"
 	AKSAzureLinuxPublicGalleryURL = "AKSAzureLinux-f7c7cda5-1c9a-4bdc-a222-9614c968580b"
+
+	AKSUbuntuResourceGroup     = "AKS-Ubuntu"
+	AKSAzureLinuxResourceGroup = "AKS-AzureLinux"
+	AKSWindowsResourceGroup    = "AKS-Windows"
+
+	AKSUbuntuGalleryName     = "AKSUbuntu"
+	AKSAzureLinuxGalleryName = "AKSAzureLinux"
+	AKSWindowsGalleryName    = "AKSWindows"
```

Comment:
We aren't using the consts here, but figured I would add them since I was looking at these gallery definitions, and we don't want to have to go through tracking them down again. 

I can remove them if we decide we don't want anything unused, but figured rather than making someone solve that in the future I should just note it down here while i touch this bit.